README.md

```diff
@@ -114,3 +114,7 @@ Replace the first two steps above with:
 
 1. Install [`mdbook_katex_css_download`](https://github.com/SichangHe/mdbook_katex_css_download).
 1. Run the above executable at the root of your mdBook project.
+
+## Integrated one-click setup (Windows-only)
+1. Clone the repository on Windows
+2. Run `build.vbs` to build the book. The mdbook and mdbook-katex binaries as well as KaTeX css and fonts will be downloaded automatically if they are not present.
```

Comment:
Is it possible to separate the binary download (mdbook & mdbook-katex) and the assets download (CSS & fonts)?

Reply:
Could you spilt the one `.vbs` file into two: one downloads the executable, the other downloads the assets?

This is good for now, though. I understand that these scripts are tricky.